install.md

```diff
@@ -1,104 +1,52 @@
 
-_The [installation guide](http://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html) contains more detailed instructions_
+Which one you use is up to you. Pipenv is recommended by the [Python Packaging Authority](https://www.pypa.io/). Conda and mamba are preferred by many members of [PyData](https://pydata.org/) and can by acquired via [miniforge](https://github.com/conda-forge/miniforge#mambaforge).
 
-### Installation with mamba or conda
+## JupyterLab
 
-JupyterLab can be installed with `mamba` and `conda`:
+Install JupyterLab with one of the following:
 
 ```bash
+pipenv install jupyterlab
 mamba install -c conda-forge jupyterlab
-```
-
-or
-
-```bash
 conda install -c conda-forge jupyterlab
 ```
 
-Note: If you have not installed mamba or conda yet, you can get started with the [miniforge](https://github.com/conda-forge/miniforge#mambaforge) distribution.
-
-### Installation with pip
-
-If you use `pip`, you can install it with:
-
-```bash
-pip install jupyterlab
-```
-
-If you are using a system such as macOS that includes both Python 2 and Python 3, run `pip3` instead of `pip`.
-
-If installing using `pip install --user`, you must add the user-level `bin` directory to your `PATH` environment variable in order to launch `jupyter lab`. If you are using a Unix derivative (e.g., FreeBSD, GNU/Linux, macOS), you can do this by running ``export PATH="$HOME/.local/bin:$PATH"``.
-
-### Run JupyterLab
-
 Once installed, launch JupyterLab with:
 
 ```bash
 jupyter-lab
 ```
 
-## Getting started with the classic Jupyter Notebook
-
-### Installation with mamba or conda
-
-The classic notebook can be installed with `mamba` and `conda`:
-
-```bash
-mamba install -c conda-forge notebook
-```
+## Jupyter Notebook
 
-or
+Install the classic Jupyter Notebook with one of the following:
 
 ```bash
+pipenv install notebook
 conda install -c conda-forge notebook
+mamba install -c conda-forge notebook
 ```
 
-### Installation with pip
-
-If you use `pip`, you can install it with:
-
-```bash
-pip install notebook
-```
-
-Congratulations, you have installed Jupyter Notebook! To run the notebook, run the following command at the Terminal (Mac/Linux) or Command Prompt (Windows):
+To run the notebook:
 
 ```bash
 jupyter notebook
 ```
 
-See [Running the Notebook](https://jupyter.readthedocs.io/en/latest/running.html#running) for more details.
-
-## Getting started with Voilà
+## Voilà
 
-### Installation with mamba or conda
-
-If you use `mamba` or `conda`, you can install it with:
+Install Voilà with one of the following:
 
 ```bash
+pipenv install voila
 mamba install -c conda-forge voila
```

Comment:
Ugh I can't suggest edits for lines that were removed - if you agree w/ the change we can manually remove the extra lines